pyproject.toml

```diff
@@ -33,13 +33,17 @@ include = [
 [tool.poetry.dependencies]
 python = "^3.6.2"
 # Background task processing
-celery = "~5.1.0"
+celery = "~5.1.2"
+# Prevent duplicate celery tasks
+celery_once = "~3.0.1"
 # Fundamental web framework for Nautobot
 Django = "~3.1.12"
 # Caching with Redis
 django-cacheops = "~5.1"
 # Base functionality for task scheduling with Celery
 django-celery-beat = "^2.2.0"
+# Management of app configuration via the Django admin UI
+django-constance = {version = "~2.8.0", extras = ["database"]}
```

Comment:
Looks like `next` has gotten a bit behind `develop`, but I think this PR should be targeted to `develop` rather than `next` at this point?

Reply:
Yes. I'll fix that.